core/src/test/scala/org/apache/spark/sql/delta/DeltaTimeTravelSuite.scala

```diff
@@ -737,21 +737,18 @@ class DeltaTimeTravelSuite extends QueryTest
     }
   }
 
-
-  // This is currently set up to fail once the bug is fixed (should be when upgrading to Spark 3.4)
-  // See https://github.com/delta-io/delta/issues/1479
-  test("SPARK-41154: Incorrect relation caching for queries with time travel spec") {
+  test("SPARK-41154: Correct relation caching for queries with time travel spec") {
     val tblName = "tab"
     withTable(tblName) {
       sql(s"CREATE TABLE $tblName USING DELTA AS SELECT 1 as c")
       sql(s"INSERT INTO $tblName SELECT 2 as c")
-      assert(
+      checkAnswer(
         sql(s"""
-              |SELECT * FROM $tblName VERSION AS OF '0'
-              |UNION ALL
-              |SELECT * FROM $tblName VERSION AS OF '1'
-              |""".stripMargin
-        ).collect() === Array(Row(1), Row(1))) // this should be Array(Row(1), Row(1), Row(2))
+          |SELECT * FROM $tblName VERSION AS OF '0'
```

Comment:
why this change? we upgraded to spark 3.3.2 not 3.4?

Reply:
The fix was patched to 3.3.2 https://spark.apache.org/releases/spark-release-3-3-2.html - SPARK-41154

Reply:
Ah, makes sense. Thanks!